kitsune/users/admin.py

```diff
@@ -34,6 +35,7 @@ class ProfileAdmin(admin.ModelAdmin):
     list_display = ['full_user']
     list_select_related = True
     search_fields = ['user__username', 'user__email', 'name']
+    readonly_fields = ['is_fxa_migrated', 'fxa_uid']
```

Comment:
It might be useful, initially to be able to edit these fields for testing purposes. How about dropping this change and opening an issue instead just to keep track of this?

Reply:
Sure. I'll remove this for now